### ISSUE TITLE
Make the Travis build succeed with no errors so that we can do proper unit testing

### ISSUE THREAD

**Issue by ibell** [ibell/coolprop#108]
It is hard to tell whether we have screwed things up with the build since there are always errors in the Travis CI build.  Need to resolve/remove the failing tests.

**Comment by Rod-Persky** [comment]
So I was thinking about this! The idea was:
- When the project builds, send a list of test results somewhere such that,
- When it comes to the next build a success is determined by that test not changing state (failing)

I was looking though the travisci build system, and we can also have it automatically zip up and push binaries to pypi, sourceforge and wherever else.

**Comment by ibell** [comment]
One thing they have is a passing or failing png you can embed in a website,
a la https://github.com/philsquared/Catch (see in the readme.md)

**Comment by Rod-Persky** [comment]
If we get it to stop failing the specific tests that it does, then that png should be pretty good - otherwise i'll be deceiving that the program is completely broken.

**Comment by ibell** [comment]
Right, which is why I haven't implemented that yet.  I've been adding a lot
of tests recently at the C++ level which we can also call from python.
Should make for a pretty solid testing system.

**Comment by ibell** [comment]
@Rod-Persky Can you show me where to set up the hooks for the pypi uploading, etc?
